deploy/templates/default-azuredeploy-webjobs.json

```diff
@@ -0,0 +1,604 @@
+{
+  "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "ServiceName": {
+      "type": "string",
+      "minLength": 3,
+      "maxLength": 20,
+      "metadata": {
+        "description": "Name for the service(s) being deployed.  Name will applied to all relevant services being created."
+      }
+    },
+    "RepositoryUrl": {
+      "type": "string",
+      "defaultValue": "https://github.com/Microsoft/iomt-fhir",
+      "metadata": {
+        "description": "Repository to pull source code from. If blank, source code will not be deployed."
+      }
+    },
+    "RepositoryBranch": {
+      "type": "string",
+      "defaultValue": "master",
+      "metadata": {
+        "description": "Source code branch to deploy."
+      }
+    },
+    "JobWindowUnit": {
```

Comment:
this is for stream analytics, but I am considering also using this value to set batching windows in the stream analytics replacement. haven't complete that work yet, but will in some future commits.